tests/Agent/IntegrationTests/SharedApplications/ConsoleMultiFunctionApplicationCore/Program.cs

```diff
@@ -11,6 +11,7 @@ class Program
     {
         static void Main(string[] args)
         {
+//            System.Diagnostics.Debugger.Launch();
```

Comment:
We should probably this and any other debugging related comments.

Reply:
Definitely remove this. I'm only commenting here to plug another way to launch the debugger. You can make the first command sent to the ConsoleMF app `RootCommands LaunchDebugger` and get similar utility. 

https://github.com/newrelic/newrelic-dotnet-agent/blob/da64b94e4f2fe6fd1b65dedd29051ee8704a10f3/tests/Agent/IntegrationTests/SharedApplications/Common/MultiFunctionApplicationHelpers/RootCommands.cs#L22-L26